release/models/isis/openconfig-isis-lsp.yang

```diff
@@ -34,7 +34,14 @@ submodule openconfig-isis-lsp {
     Section 4.c of the IETF Trust's Legal Provisions Relating
     to IETF Documents (http://trustee.ietf.org/license-info).";
 
-  oc-ext:openconfig-version "1.4.1";
+
+  oc-ext:openconfig-version "1.4.2";
```

Comment:
@nokia1adam Apologies for not commenting on this earlier, but the version needs to have the minor part incremented.  ie: 1.5.1.  This is because a leaf is being added. (see [semver.org](https://semver.org/))

Reply:
Thanks Adam, 1.5.0 should be the correct one:
`Patch version MUST be reset to 0 when minor version is incremented.`